docs/usage/initialization.rst

```diff
@@ -22,6 +22,26 @@ Basic initialization which is going to work for everybody:
     northwind = pyodata.Client(SERVICE_URL, requests.Session())
 
 
+Set custom namespaces
+---------------------------------
```

Comment:
Please add a blank like below `----` and cut the line `-----` to the length of the caption.